### PR TITLE
feat: support for running a given test-sets

### PR DESCRIPTION
**Pull request by AkashKumar7902**
## Related Issue
  - Info about Issue or bug

Closes: #952

#### Describe the changes you've made
Added a testsets flag for adding an array of testsets to run. Changed the Test function signature to send the testsets array passed by user. used a map of sessions to check whether the provided testset is valid or not. 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Please let us know if any test cases are added

Please describe the tests(if any). Provide instructions how its affecting the coverage.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
N/A

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.

## Screenshots (if any)

|        Original         |          Updated           |
|:-----------------------:|:--------------------------:|
| **image** | <b>image</b> |

### CODE REVIEW

File: pkg/service/test/test.go [hunk 1]
```diff
@@ -119,7 +119,22 @@ func (t *tester) Test(path, testReportPath string, appCmd, appContainer, appNetw
...
+	}
+
+	sessionsMap := map[string]string{}
+
 	for _, sessionIndex := range sessions {
+		sessionsMap[sessionIndex] = sessionIndex
+	}
+
+	for _, sessionIndex := range testsets {
+		if _, ok := sessionsMap[sessionIndex]; !ok {
```
[the remark this answers is not in the document]
Comment (AkashKumar7902) [reply pkg/service/test/test.go:134]: Sure

**Review by re-Tick — APPROVED** [review]
The changes looks good to me. Please resolve the merge conflicts.

**Comment by AkashKumar7902** [comment]
@re-Tick Sorry, i made a mess. I am too bad at git. Should i make a new PR ?

**Comment by re-Tick** [comment]
Hi @AkashKumar7902, this PR would be beneficial for tracking, we can merge this PR.

**Pull request merged**